README.md

```diff
@@ -115,14 +145,14 @@ This makes it much easier to use promises in node tests. Instead of having to re
 
 See how much simpler that is? You won't need to add `done` to the `it` or accidentally put `done` in the `describe` and wonder why your tests are failing when you've been writing tests all day.
 
-[Code Coverage]: https://codecov.io/github/tests-always-included/jasmine-node-helpers?branch=master
-[codecov-image]: https://codecov.io/github/tests-always-included/jasmine-node-helpers/coverage.svg?branch=master
-[Dev Dependencies]: https://david-dm.org/tests-always-included/jasmine-node-helpers/master#info=devDependencies
-[devdependencies-image]: https://david-dm.org/tests-always-included/jasmine-node-helpers/master/dev-status.png
-[Dependencies]: https://david-dm.org/tests-always-included/jasmine-node-helpers/master
-[dependencies-image]: https://david-dm.org/tests-always-included/jasmine-node-helpers/master.png
-[Jasmine]: https://jasmine.github.io/
+[Code Coverage]: https://codecov.io/github/tests-always-included/jasmine-test-helpers?branch=master
```

Comment:
Agreed, though we should rename the project and publish this as jasmine-test-helpers and possibly unpublish the old module.